repo2docker/utils.py

```diff
@@ -391,3 +391,24 @@ def copytree(
     if errors:
         raise Error(errors)
     return dst
+
+
+# Code segments below from idutils (https://github.com/inveniosoftware/idutils)
+# Copyright (C) 2015-2018 CERN.
+# Copyright (C) 2018 Alan Rubin.
+# Licensed under BSD-3-Clause license
+doi_regexp = re.compile(
+    "(doi:\s*|(?:https?://)?(?:dx\.)?doi\.org/)?(10\.\d+(.\d+)*/.+)$", flags=re.I
+)
+"""See http://en.wikipedia.org/wiki/Digital_object_identifier."""
```

Comment:
can we turn this into a comment preceding the regex. This isn't a doc string really/it isn't customary to have doc strings for variables.